content/docs/deployment.md

```diff
@@ -208,6 +239,37 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
+### Kafka
+Supported in Jaeger since 1.6.0
+Supported Kafka versions: 0.8+
+
+Currently, the Kafka storage backend implementation only supports writing data, this means you will need to use another one (with the multiple storage types feature) to be able to view the traces in the Query component.
+Writing to Kafka is particularly useful for building post-processing data pipelines.
+
+In a future release, there will be a new mechanism to read the data from Kafka and save it in Elasticsearch or Cassandra.
+
+#### Configuration
+##### Minimal
+```sh
+docker run \
+  -e SPAN_STORAGE_TYPE=kafka \
+  -e KAFKA_BROKERS=<...> \
+  -e KAFKA_TOPIC=<...> \
+  jaegertracing/jaeger-collector
+```
+
+##### All options
+To view the full list of configuration options, you can run the following command:
+```sh
+docker run -e SPAN_STORAGE_TYPE=kafka jaegertracing/jaeger-collector:1.6 --help
+```
+
+#### Topic & partition
+Unless your Kafka cluster is configured to automatically create topics, you will need to create it ahead of time. You can refer to [the Kafka quickstart documentation](https://kafka.apache.org/documentation/#quickstart_createtopic) to learn how.
+
+You can also find more information about topics and partitions in the [official documentation](https://kafka.apache.org/documentation/#intro_topics).
```

Comment:
Perhaps we should link to https://www.confluent.io/blog/how-to-choose-the-number-of-topicspartitions-in-a-kafka-cluster/ for selecting a reasonable number of partitions when creating the topic?

I'm a bit conflicted on whether we should even include partitions in this section of the documentation because someone operating Kafka should be pretty familiar with the concept.

Reply:
I think every extra bit helps, +1 to link to a blog

Reply:
especially if you (@vprithvi) found it useful when connecting Jaeger to Kafka